### PR TITLE
add new fields to sign up page, edit event and rewrite test

### PR DESCRIPTION
**Pull request by SetterVet**
image
image

### CODE REVIEW

**Comment by SetterVet** [comment]
@afurm @rubycop @VasylYaremkiv

**Comment by SetterVet** [comment]
image

File: spec/features/socialmedia_login_spec.rb [hunk 1]
```diff
@@ -65,17 +65,22 @@ def when_i_successfully_sign_up
     select(project.name, from: 'Project')
     fill_in 'Password', with: '123123'
     fill_in 'Password confirmation', with: '123123'
+    fill_in 'Position', with: 'position'
+    fill_in 'Birthday', with: Time.zone.today
```
Comment (VasylYaremkiv) [inline spec/features/socialmedia_login_spec.rb]: `Date.today`

Comment (SetterVet) [reply spec/features/socialmedia_login_spec.rb]: @VasylYaremkiv 
image

**Comment by SetterVet** [comment]
[File.mkv.zip](https://github.com/CyberCraftInc/cyber-dashboard/files/2844944/File.mkv.zip)

**Pull request merged**